manifests/server.pp

```diff
@@ -43,7 +43,12 @@
 ) {
 
   if $scenario_map.has_key($scenario) {
-    include simp::knockout(union($scenario_map[$scenario], $classes))
+    include simp::knockout(
+      union(
+        $scenario_map[$scenario], 
+        $classes
+      )
+    )
```

Comment:
This is style more than best practices, something more for readability and appearances than enforcing some best-practice. That said, I'm not apposed to this particular change in style, because I think it's pretty neutral.

One thing to note, however, is that the final parameter should include the trailing comma, because it aids code management best practices. Namely, it keeps diff sizes smaller and easier to read, which makes code review easier-- a best practice.